public/app/types/unified-alerting.ts

```diff
@@ -28,7 +28,7 @@ interface RuleBase {
 }
 
 export interface AlertingRule extends RuleBase {
-  alerts: Alert[];
+  alerts?: Alert[];
```

Comment:
made `alerts` optional since the "inactive" alerts seem to have `undefined` alerts, I should dig deeper in to this 🤔